internal/logger/securityevent_logger.go

```diff
@@ -79,6 +79,28 @@ func LogGrantJimmAdmins(ctx context.Context, identityIds []string) {
 	})
 }
 
+// LogUserCreated
+func LogUserCreated(ctx context.Context, identityId string) {
+	logSecurityEvent(ctx, securityEvent{
+		Event:       fmt.Sprintf("user_created:%s", identityId),
+		Description: fmt.Sprintf("User %s was created.", identityId),
+		Severity:    warning,
+	})
+}
+
+// LogUserUpdated
+func LogUserUpdated(ctx context.Context, adminID string, userID string, relation string, target string, isAddition bool) {
+	action := "adding"
+	if !isAddition {
+		action = "removing"
+	}
+	logSecurityEvent(ctx, securityEvent{
+		Event:       fmt.Sprintf("user_updated:%s,%s,%s:%s", adminID, userID, relation, target), // ?
```

Comment:
is this format correct?
the spec isn't really clear on this.. i would suggest something along these lines
```
user_updated:<admin user>,<user id>,<add/remove>,<relation>:<target>
```